internal/pkg/errors/error_message.go

```diff
@@ -495,6 +492,7 @@ const (
 	// General
 	DeleteResourceErrorMsg        = `failed to delete %s "%s": %v`
 	DeleteResourceConfirmErrorMsg = `input does not match "%s"`
+	ListResourceSuggestions       = "List available %ss with `%s list`."
```

Comment:
```suggestion
	ListResourceSuggestions       = "List available %ss with `confluent %s list`."
```

Reply:
The function that returns the full parent command includes `confluent`, so I left it off of this string.